pkgs/applications/misc/ubpm/install.patch

```diff
@@ -0,0 +1,34 @@
+diff --git a/sources/mainapp/mainapp.pro b/sources/mainapp/mainapp.pro
```

Comment:
should this be reported upstream ?

Reply:
Could be, but I'm not an expert in qmake, so I'm not sure how to properly fix it.

Reply:
Its not mandatory. Just  checking

Reply:
I will report it, but not sure if it's going to be fixed and how :)

Reply:
https://codeberg.org/LazyT/ubpm/issues/82